streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -272,4 +249,18 @@ public boolean hasStateStores() {
     public Collection<TopicPartition> changelogPartitions() {
         return stateMgr.changelogPartitions();
     }
+
+    long committedOffsetForPartition(final TopicPartition partition) {
+        try {
+            final OffsetAndMetadata metadata = consumer.committed(partition);
```

Comment:
@guozhangwang With regard to you comment about exceptions for EOS -- it seem only relevant to catch and swallow a `TimeoutException` here?

Reply:
I think TimeoutException is the one, yes, but more importantly we should see if our timeout values are set reasonably in Streams :)